package.json

```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://github.com/mate-academy/api_movies#readme",
   "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
there are `express.json()` method which return `body-parser` middleware, no need to install it separately